stories/assets/js/tabs.js

```diff
@@ -4,18 +4,27 @@
  * Finds all tabs on a page and activates them
  * @param {object} [scope] - the html scope to process, optional, defaults to `document`
  * @param {boolean} [activateDeepLinkOnLoad] - if deep linked tabs should be activated on page load, defaults to true
- * @example vfTabs(document.querySelectorAll('.mg-component__container')[0]);
+ * @example mgTabs(document.querySelectorAll('.mg-component__container')[0]);
  */
-function mgTabs(scope, activateDeepLinkOnLoad) {
+export function mgTabs(scope, activateDeepLinkOnLoad) {
   /* eslint-disable no-redeclare */
   var scope = scope || document;
   var activateDeepLinkOnLoad = activateDeepLinkOnLoad || true;
   /* eslint-enable no-redeclare */
   // Get relevant elements and collections
   const tabsList = scope.querySelectorAll("[data-mg-js-tabs]");
   const panelsList = scope.querySelectorAll("[data-mg-js-tabs-content]");
+
   const panels = scope.querySelectorAll("[data-mg-js-tabs-content] [id^=\"mg-tabs__section\"]");
   const tabs = scope.querySelectorAll("[data-mg-js-tabs] .mg-tabs__link");
+
+  console.log("debug: All panels");
```

Comment:
Forgot to comment some console.log commands

Reply:
yeah yeah. good catch